### ISSUE TITLE
annotate a MF activity unit on a specific sub-part of a complex

### ISSUE THREAD

**Issue by ValWood** [geneontology/noctua#901]
We would like to be able to  annotate a MF activity unit on a specific sub-part of a complex

(discussed with @vanaukenk , but I don't think there is a ticket for this)

**Comment by ValWood** [comment]
As an aside, I'm not sure that this is really "positive regulation" it's definitely "positively affects", but I think the labels on the arrows are sometimes misleading. @vanaukenk  what would you use in this context?

**Comment by thomaspd** [comment]
The current guidelines are to associate the function with the complex only if it's an "emergent" function that can't be ascribed to a single gene product, which doesn't seem to be the case here, at least to some degree of approximation. I think it's best for users if we can associate functions with individual gene products whenever possible. If you want to say that ndc80 is in this complex when it's active, there's been a proposal to allow occurs_in (protein complex GO term) to be added to the current spec, in addition to allowing location to be specified by cellular anatomical entity and cell type. Would that work for this case?

**Comment by ValWood** [comment]
Well its that old problem of "molecular machines" I guess. 

I think these complexes do have "emergent functions, but we haven't figured out how to describe them. How  do we describe this in GO?
![lateral attachemnt](https://github.com/user-attachments/assets/594639d3-8e77-4313-a310-d65df25e682c)
![kinetochore-splindle](https://github.com/user-attachments/assets/5381582b-b151-42bb-831e-f113934d874b)
<img width="739" alt="microtubule kinetochore attachment" src="https://github.com/user-attachments/assets/acda10e3-55de-41ad-8f1e-0859dffb07d8">


We have activities annotated to specific subunits (because this is the  nature of the experimental  data and we don't want to lose that detail)
 but i) we still want to represent all of the complex members in the model

For example ndc80 is the adaptor for mph1 and alp7 based on experiment, but clearly the entire complex contributes to the function and we don't want to leave other subunits  out of the model or it will be incomplete.

We are left with 2 options
1, use a complex and lose the  individual subunits
This would lose the experimental detail

2. Add the individual subunits but not grouped as a complex (this will look incredibly messy for  kinetochore pathways  in the go-cam-viz)

**Comment by ValWood** [comment]
We haven't modelled the kinetochore pathways yet because we are waiting on the complex import issues.

But we find similar in the  various heterochromatin pathway modelling:

See for example:
http://noctua.geneontology.org/workbench/noctua-visual-pathway-editor/?barista_token=sasgrjd2nx44ugmbfn3w&model_id=gomodel:665912ed00000652


The go-cam viz looks like this :
<img width="586" alt="Screenshot 2024-09-06 at 17 33 41" src="https://github.com/user-attachments/assets/47c3757e-f8d3-4471-be79-5c9383b63e94">


(Note that not all components are rendering correctly, some connections are missing)

This is really a display issue, rather than a curation issue. It is hard for users to  to navigate  but would look much better if we could group  activities by complex in the viewer. It would make much more biological sense. Especially as these pathways become increasingly  complicated.

<img width="288" alt="Screenshot 2024-09-06 at 17 30 32" src="https://github.com/user-attachments/assets/b0c199a0-5f79-4b7e-9d24-369d30616ca0">

(note that each complex subunit  usually has its own annotation, in addition to being part of the complex, which could be described as having an emergent function like "chromatin remodelled")

**Comment by thomaspd** [comment]
Yes, it's a tough call on when to use the complex as the functional unit. Judging from the Fig 4b above, you can't assign the dam1/dash adaptor activity to just ndc80 since it seems to also depend on nuf2. So it seems like you could subdivide the complex into two distinct complexes, ndc80/nuf2 (dam1/dash to spc24/25 adaptor) and spc24/25 (ndc80 to mind adaptor). But since these two components ndc80/nuf2 and spc24/25 appear to always be in a larger complex together, and both are required for the overall activity, it seems to make sense to me to make the entire ndc80 complex a functional unit. If you want, we can include the details of what each subunit does, in the definition of the ndc80 complex GO term.

**Comment by ValWood** [comment]
That wouldn't work in the heterochromatin example  above though....and we would lose a lot of detail that we would want to include in the model.  i.e the annotations are on the individual subunits. Sometimes these capture complex modification dependencies etc.

We could just lump activities onto the complex but then we could not reuse existing annotations (which we really want to do),

For example CLRC (image above) has 3 adaptors, a UB ligase and a methyltransferase , and a subunit of unknown function.
RITs  (image above) have an endonuclease and a histone reader.
These pathway are going to be very difficult to navigate, and harder for users to understand  unless we can organize by complex in the viewer. The user experience would be much better if we represented the complex in the graphviz, and the user could use the "expand protein complexes" button to see the activities attached to the individual subunits.

**Comment by thomaspd** [comment]
The chromatin remodeling complexes sound like they're pretty modular-- each subunit has a distinct activity and we should try to describe the causal relations between those activities in the GO-CAM. The discussions we've been having with Colin about chromatin remodelling have been helpful, and you should attend the next one. @pgaudet can you please make sure Val is invited to our next discussion with Colin?

**Comment by ValWood** [comment]
We have described the individual activities (you can see them in the model above). It's just hard, even for an expert to consume the information in the display without grouping by complex (because this is how they think about, and draw, the pathways.
 
Actully @PCarme did our chromatin remodelling pathways and he is pretty expert in this (and pombe probably has the most detail in this area)

The pathway above is centromeric chromatin remodelling, and we also have mating type region and subtelomeric, the pathways are slightly differnt.

**Comment by ValWood** [comment]
I was attending the Chromatin calls until relatively recently but I had to drop off because too much on. I can try to attend on Friday but  it depends if I am finished up with other stuff before my hol the week after and next week is already pretty rammed with meetings. Pascal could join but he  is also away next week. Maybe we can start to attend after holidays PAscal  is back before me so if you could CC him he could probably attend.

**Comment by Antonialock** [comment]
@Pauldenny

**Comment by thomaspd** [comment]
@ValWood do we need further discussion or can we close this issue?

**Comment by ValWood** [comment]
We still need to be able to do it. I can provide examples and why.

**Comment by ValWood** [comment]
This will be resolved by the ability to add part_of complex relations to entities.
Is there another ticket for this?

**Comment by Pauldenny** [comment]
Hi @ValWood @vanaukenk @thomaspd - is the ability to add a part_of complex relation to entities already planned or simply a suggestion?

**Comment by ValWood** [comment]
It was agreed that it  will be possible to say that something is a subunit of a complex in Noctua.

Inthe minutes
Document decision to capture complex contextual information as ‘part of’ relation between ‘Active Entity’ -> ‘Part of’
https://docs.google.com/presentation/d/1vF8JCnUM_JqkxgAvVHHl6ntyXpk3tY4y_NAjRjZ3WJE/edit?slide=id.p#slide=id.p
v

**Comment by thomaspd** [comment]
Yes, this is now on the list of official feature requests to be implemented.

**Comment by vanaukenk** [comment]
See this ticket in the VPE:  https://github.com/geneontology/noctua-visual-pathway-editor/issues/121

**Comment by vanaukenk** [comment]
@ValWood @thomaspd - there's a lot of good discussion here, but the request for the functionality is on the VPE tracker.  Should we close this ticket, or move it to the VPE tracker?

Thanks.

**Comment by pgaudet** [comment]
Duplicate of https://github.com/geneontology/noctua-visual-pathway-editor/issues/121
